inc/settings.php

```diff
@@ -125,7 +125,7 @@ function siteorigin_north_settings_init(){
 				'sidebar_width' => array(
 					'label'             => __( 'Sidebar Width', 'siteorigin-north' ),
 					'type'              => 'measurement',
-					'live'              => true,
+					'live'              => false,
```

Comment:
It seems like a waste to lose live previewing to deal with this edge case. Maybe we'd be better off just implementing some sort of maximum and minimum value options for measurement fields (if they don't already exist).